package.json

```diff
@@ -17,7 +17,9 @@
     "prepare": "npm run compile",
     "pretest": "npm run compile",
     "codecov": "nyc report --reporter=json && codecov -f coverage/*.json",
-    "commitlint": "git log -1 --pretty=%B | commitlint"
+    "commitlint": "git log -1 --pretty=%B | commitlint",
+    "webpack": "webpack",
+    "browser-test": "sh browser-test/browser-test.sh"
```

Comment:
Would you mind adding a new job to the CI config for this as well?  Seems like the kinda thing that should run on PR.